### DISCUSSION TITLE
刷牙 | /posts/brush-teeth/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#575] [Announcements]
## 刷牙

我的思绪总是在擦拭我的童年，直到它变得像一块坚硬的钻石，不可破碎、切入，我成年的廉价玻璃。—-耶胡达·阿米亥《夏末黄昏在摩查》

https://yuanfan.rbind.io/posts/brush-teeth/

**Comment by hongtaoh** [comment]
我记得之前看过一篇文章，它说刷牙的时候要保持口腔干燥，不然好像无法发挥出牙膏的效用，所以刷牙前最好不要漱口。近十年我都是这么干的。

**Reply by earfanfan** [reply]
我今天白天和晚上都试了试直接刷牙，干刷最开始会有一点涩的感觉，但是刷完牙还是咕咚咕咚了的。话说我上大学的时候，我们宿舍老大说要用反手刷牙，好像是这样会更聪明还是撒，于是作为右撇子的我们试着用左手刷牙，后来我觉得始终不习惯，坚持不下去就放下了。

**Reply by hongtaoh** [reply]
哈哈。我从初中开始就左手刷牙，已经 17 年了。

**Reply by earfanfan** [reply]
自从知道左手刷牙是可以长期做到以后，我这两个月99%都是用左手刷牙。

**Reply by CyrusYip** [reply]
接下来是不是要挑战左手吃饭，哈哈。

**Comment by yihui** [comment]
活雷锋路过：

> My thoughts are always polishing my childhood
> Till it’s become like a hard diamond
> Unbreakable, to cut
> Into the cheap glass of my maturity
>
> ---Yehuda Amichai

《醋栗》中的名段落是指鸫鸟南飞的那段吗？

> 我们完全跟农家孩子一样，白天晚上都待在田野上、树林里，看守马匹，剥树皮，捕鱼，以及诸如此类的事情……你们也知道，谁一生中哪怕只钓到过一条鲈鱼，或者在秋天只见过一次鸫鸟南飞，看它们在晴朗凉爽的日子成群飞过村子，那他已经不算是城里人，他至死都会向往这种自由的生活了。

这篇小说的结尾我觉得写得非常好，就是一句话：

> 雨整夜敲打着窗子。

中间有两段伊凡 · 伊凡内奇的议论也很有力量：

> 显然，幸福的人之所以感到幸福，只是因为不幸的人们在默默地背负着自己的重担。一旦没有了这种沉默，一些人的幸福便不可想象。这是普遍的麻木不仁。真应当在每一个心满意足的幸福之人的门背后，站一个人，拿着小锤子，经常敲门提醒他：世上还有不幸的人；不管他现在多么幸福，生活迟早会对他伸出利爪，灾难会降临——疾病，贫穷，种种天灾人祸。到那时，面对他，谁也视而不见、听而不闻——现在他不也是看不见别人、听不见别人吗？可是，拿锤子的人是没有的，幸福的人照样过他的幸福生活，只有日常生活的小小烦恼才使他感到有点儿激动，就像微风吹拂杨树一样。一切都幸福圆满。

> 自由是好东西，我也这样说，没有自由就像没有空气一样是不行的，但目前还得等待。是的，我就是这样说的，不过我现在要问：为什么要等待？”伊凡·伊凡内奇生气地望着布尔金，问道，“请问，为什么要等待？出于什么考虑？别人对我说，凡事不能一蹴而就，任何理想总是在生活中逐步地、在适当的时候实现的。不过，这是谁说的？有什么证据说明这是对的？你们会引证事物的自然规律和社会现象的合法性。但是我请问：我，一个有思想的活人，站在一道沟前，本来我也许可以跳过去，或者在上面架一座桥走过去，我却偏要等着它自己合拢，或者等着淤泥把它填满，这样做有什么规律和合法性可言？再说一遍，为什么要等待，等到没法活的那一天吗？可是人需要生活，渴望生活！

《竹峰寺》的语言让我想起有次看羽毛球比赛时听到的解说，解说员说高水平运动员到最后拼的还是天赋。陈春成这个古文天赋确实是太强了。

**Reply by earfanfan** [reply]
我看《醋栗》就是最早从你的博客读到那个段落，然后找来全文去读的，那个时候读到的感想和你现在是一样的，但我今天又插队重读了一遍，我的观念已经变了。

>我看见了一个幸福的人，他的心心念念的梦想显然已经实现，他的生活目标已经达到，他所想望的东西已经到手，他对他的命运和他自己都满意了。不知什么缘故，往常我一想到人的幸福，就不免带一点哀伤的感觉，这一回亲眼看到幸福的人，我竟生出一种跟绝望相近的沉重感觉。
>
>我心想：实际上有多少满足而幸福的人啊！这是一种多么令人沮丧的势力！你们看一看这种生活吧：强者骄横而懒惰，弱者无知而且跟牲畜那样生活着，处处都是叫人没法相信的贫穷、拥挤、退化、酗酒、伪善、撒谎……可是偏偏所有的屋子里也好，街上也好，却一味的心平气和，安安静静。一个城市的五万居民当中竟没有一个人叫喊一声，大声发泄一下他的愤慨。我们看见人们到市场上去买食物，白天吃饭，晚上睡觉，他们说废话，结婚，衰老，心平气和地送死人到墓园去。可是那些受苦受难的人，那些在幕后什么地方正在进行着的人生惨事，我们却没看见，也没听见。处处都安静而太平，提抗议的只有那些没声音的统计表：若干人发了疯，若干桶白酒喝光了，若干儿童死于营养不良……这样的世道显然是必要的，幸福的人所以会感到逍遥自在，显然只是因为那些不幸的人沉默地背着他们的重担，缺了这种沉默想要幸福就办不到。这是普遍的麻木不仁。

记得之前有一次向你提过我打算读张秋子老师的《堂吉诃德的眼镜》，不过后来其实我让别的书插队了，至今还没开始读。不过今天重读醋栗的时候，发现有人留言说这本书里提到了醋栗，就把跟醋栗有关的段落找出来读了一下，主要是“第五讲 《幸福》：文学中的意义感”，然后我就很惊讶地发现，原来张秋子老师写的内容正好就是我最近在思考的。真得，我昨晚刚又把马斯洛的那个需求层次理论翻出来看了一下，最高一层的自我实现真得是每个人都应该去追求的吗？而今晚看张秋子老师的书，她也提到类似的观点，正好就是我想过的，我把她的写在下面：

>契诃夫对人的要求会不会太高了？这是我近些年思考的问题……成为一个平庸之辈、过着平庸的生活就应该受到讽刺？当我们说追逐社会等级和物质利益是粗俗的时候，为什么说追求精神世界就显得很优越呢？就像大家在解读《幸福》时说伯莎有钱而空虚，没有精神追求，好像一个人有了精神世界就更高级一些。这些年，随着年龄的增长，我会比较警惕以打压日常平庸带来的精神优越感，因为，它可能同样是被填入的意义感。

在她写这一段之前，有解释一下契诃夫感到绝望的原因：

>什么是自动让淤泥填满壕沟？契诃夫是指，我们自动地接受了生活的意义，让外在的、被指定的意义感填满了内在世界。这是他觉得绝望的原因，因为，一个有思想的活人应该自己去创造生活的意义，而不是被动地、不假思考地让外在赋予的意义感“填入”自身，甚至为此感到幸福不已。这种不经省察的幸福，在苛刻的契诃夫看来，是庸俗乏味的。

其实，我渐渐明白到，其实每个人都会思考，都拥有自己的精神世界，因为人都有情感、都有生命。诚然，我对自己的要求和契诃夫是一样的，就是睁开眼睛去看见、用心去观察世界，而不是被动接受其他任何人或事物对我的思想灌溉。但我不觉得应该要求每个人都和我一样，而是应该尊重每个人的想法，即便与我不同。我越来越喜欢与朋友们交流分歧了，可是益辉你给我的感觉始终像是沉浸与找到相同，像是在每个人身上去找到你的分身。我曾经想过一个问题，当你认为我是你的朋友时，是不是只接受了我的那些天真、那些在你看来是你没有但我拥有的被你视作珍贵的品质，哈哈，那是很早以前想的，我想现在的你应该早就接受了事实，你的朋友是多么平凡普通。在我的眼里，你也是一个平凡普通的人类，顶着科学家职业身份的三水也是，大家都是有优点也有缺点的普通人。

这里我吐一个槽，一个很久以前就记在心里的，好像是在你的博客里面曾经提到过一个细节，大概就是你有一个亲戚生病住院了，你给了她五千块，她对你千恩万谢、感激涕零，但是在你的眼里觉得五千块不算什么，根本用不着她那样地感谢你。我觉得你忽略了她的一种很质朴的品质，尽管在她看来五千块是算多的，她会因为这么数字感谢你，但是你记得她给她帮助也是她想感谢的地方啊，尤其是在生病的时候，任何关怀包括金钱的形式都是雪中送炭。

我记得有一次我跟鸿涛在我的博客留言区讨论时提到，益辉你通过自己的努力跨越了阶级，后来你回复说你与我们同在一个阶级。跨越阶级的标准当然可以讨论，你的心当然可以与我们同在一个阶级，但是有些方面也许不是，如果你还想和我们待在一个阶级的话，恐怕你要开始努力了，因为你的日常生活里已经很难接触到、或者看到、或者真切地感受到人们为求生存而经历的苦难了。你可以通过互联网看新闻，但是新闻毕竟难以融入你的日常生活，就如同别人的苦难看多了会为你带来精神上的苦痛，因为你只能袖手旁观，或者漠视。

**Reply by earfanfan** [reply]
>《竹峰寺》的语言让我想起有次看羽毛球比赛时听到的解说，解说员说高水平运动员到最后拼的还是天赋。陈春成这个古文天赋确实是太强了。

陈春成古文天赋这个何解？

**Reply by earfanfan** [reply]
话说这次看《醋栗》，我特意慢慢地看，发现有不少细节很有些意思。

>看见远处一列火车，像是毛毛虫在爬。

这种远处长条状物体像毛毛虫的感觉我也有过，如之前在博客里写的：“田野上有一些长得很茂密的树，树儿们被栽种地很紧密，它们挤在一起，油绿的枝叶互相包裹对方，从很远的地方看像是一条绿色的毛毛虫伏在地上。”

>> 他们往斜下里拐过去，穿过已经收割过的田地，时而照直走，时而往右走，后来走到大道上了。不久出现了白杨和花园，后来出现了谷仓的红房顶。有一条河，河水闪闪发光，于是眼界豁然开朗……

这段也写的很好，契诃夫用了“出现了”而不是“看见了”，“出现”让我联想到的是那些景物是随着地平线往前移动而从地平线上方慢慢冒出头来，一点点出现的。当然啦，这里，我想象不到闪闪发光的河水长撒样，明明他们是因为下雨要去躲雨才往那边走，下着雨的河水，应该是被雨点砸出很多小水坑的，怎么会闪闪发光，除非那个时候雨停了，太阳出来了。

>伊万·伊万内奇和布尔金走进那所房子，遇到一个使女，是个年轻女人，长得很美，他俩一下子都站住，互相瞧了一眼。

这里两个男人看到一个美女，互相瞧一眼是撒意思，确认一下眼神，都觉得年轻的使女很美丽么？

>他把水搅起波浪，弄得白色的百合在水浪上摇摇摆摆。

这里我对白色的百合疑惑不解，怀疑是翻译的问题，我看的是微信读书上面《契诃夫小说选集·醋栗集》。

> 弟弟尼古拉坐在他那办公室里，梦想将来怎样喝他自己家里的白菜汤，那种汤怎样散发满院子的清香，他怎样在绿草地上吃饭，怎样在太阳底下睡觉，怎样一连好几个钟头坐在大门外的凳子上眺望田野和树林。农艺书和日历上所有那些农艺建议，成了他的欢乐，成了他心爱的精神食粮。

其实我不知道我的观点会不会是另一种“何不食肉糜”，因为我是能回去的，回去感受田野的，但是益辉你是很多年不曾回去的，当我在批判你的时候是不是也犯了这样的错误，那是我拥有的你无法拥有的……

>到处都是沟渠、围墙、篱笆、栽成一行行的杉树，弄得人不知道怎样才能走到院子里去，应该把马拴在哪儿。我向房子走去，迎面遇见一条红毛的肥狗，活像一头猪。它想叫一声，可又懒得叫。厨娘从厨房里走出来，是一个光脚的胖女人，看样子也像一头猪。她说主人吃过饭后正在休息。我走进去看我弟弟。他在床上坐着，膝上盖一条被子。他老了，胖了，皮肉发松，他的脸颊、鼻子、嘴唇，全都往前拱出去，眼看就要跟猪那样咕咕叫着钻进被子里去了。

我看这段的时候觉得写得很好笑，不过当我意识到契诃夫是在讽刺他们像猪的时候，就觉得不好笑了。其实换个角度看，胖狗、胖女人、胖地主是不是都是因为生活安逸满足才心宽而体胖的呢。

**Reply by yihui** [reply]
> 契诃夫对人的要求会不会太高了？

我非常吃惊张秋子老师用了“优越感”这个词，我自己对契诃夫的感觉是他是很有慈悲之心的，并非想要彰显自己有多“优越”。他议论的那些问题，和迅哥儿并没有多大不同，是要召唤（麻木的？）大众多关注社会问题：贫穷、拥挤、退化、酗酒、伪善、撒谎……若干人发了疯，若干桶白酒喝光了，若干儿童死于营养不良……幸福的人之所以感到幸福，只是因为不幸的人们在默默地背负着自己的重担……

正视并试图解决这些问题，咋能说是“高要求”或追求“优越感”呢，咋能用“苛刻”来形容他呢。

另外，读者应该多大程度上把小说人物的思想与小说家本人的思想联系起来，我觉得也是个问题。通常读者都会比较自然地将作者代入小说人物，认为作者是在用小说人物为自己发声，其实也未必，这个联系不一定那样紧密。人物是塑造出来的，塑造的动机并不一定是作者自己有话要借人物之口说出来，而也许仅仅只是为了塑造一个个性鲜明的人物形象，这个形象可以代表一批人，而不仅仅（甚至不是）作者本人。这一点，其实也是我写小说的一大顾虑：我不想读者认为小说中的某个人物就是我自己，否则我的小说很可能会给我自己招惹一些很麻烦的误会。所以张秋子老师可以说伊凡内奇高要求或苛刻，但我觉得说契诃夫苛刻是不太合适的。

你提到了“讽刺”（他们像猪），这让我想到契诃夫另一篇小说《宝贝儿》，你可以瞄一眼袁哲生写的《契诃夫的尾巴》那一节（《宝贝儿》在台湾译作《可爱的女人》）：

> 契诃夫是现代短篇小说的开山鼻祖之一，他最了不起的地方便是在描写小人物的悲喜时，同时反映了广大人类的共同命运。当然，契诃夫的写作技巧也是简练如诗的，不过，因为他的艺术之眼实在令人激赏，相较之下，写作技巧便只是余事了。
> 
> 大文豪托尔斯泰为契诃夫这篇《可爱的女人》写了一则极为著名的评论。他认为契诃夫本来是要嘲弄欧莲卡的，“他用聪明而不用心灵来批评她”，“然而那个女人的忠诚，和她不管爱上谁就会献出她全心全意的忠诚，却不是荒谬的，而是奇妙的、神圣的”。托尔斯泰说，这篇伟大的小说有许多段落让他热泪盈眶。
> 
> 我认为，托尔斯泰的这个观点也应该会让契诃夫热泪盈眶。
> 
> 重点是契诃夫“本来是”要嘲弄的，他的写作技巧足以担当此任，可是写到后来他放弃了这个指指点点的“技巧”，转向“同情”与“珍惜”欧莲卡的方向去了。托尔斯泰举了一个《圣经》里的例子，先知巴兰走上山巅，准备诅咒以色列人，然而，事到临头，他却为他们祝福起来，一连三次。
> 
> 这是作家的艺术之眼真正灵光乍现的片刻。关键在于托尔斯泰所说的“聪明”与“心灵”之别。写作技巧如果依附在聪明之上时，它是可以被单独欣赏的，一如算式解答，合理而没有意外。然而好的作品需要意外，当心灵启动的时候，作品自己会“本能地”要求作者不准用“偏见”减损它应有的光彩，技巧于是只好夹着尾巴逃跑了。

在没看袁哲生的解读之前，我没太明白契诃夫对奥莲卡这个人物的态度到底是什么，我的感觉是这个女人好像有点傻傻的、神经兮兮的。读了他的解析，不得不说大受震撼，尤其是最后那句“技巧于是只好夹着尾巴逃跑了”，这句话也对我是一句很大的鼓励（因为我自认为不是技巧型的选手）。

最近几个月我有一个生活感悟就是，其实与人交往的快乐幸福感都是基于一个基本的假设，这个假设很难被证伪或证实，而是像一种信仰一样，那就是：你假设对方是怀有善意还是恶意。读小说也是如此，我们是要假设契诃夫想表达瞧不起庸俗乏味的大众生活，或是假设他怀有深刻的慈悲心？没有正确的答案，选择全在读者心里。狗、厨娘、弟弟看起来都像猪，他是想笑话他们又丑又懒，还是怜惜他们过着那样吃了睡、睡了吃的日子枉费人生？当然，我们完全可以说，过着猪一样日子也是一种选择，他们完全有权像猪一样活着、他们自己开心就好。

> 他把水搅起波浪，弄得白色的百合在水浪上摇摇摆摆。

我读的是姚锦镕翻译的《契诃夫中短篇小说选（全集）》（2015 年出版），感觉翻译水平比你读的版本要高一些，这句话在我这本里是这样的：

> 他搅起了水波，白色的睡莲便随波荡漾。

这样翻译显然简练得多，而且也准确（水里怎么会有百合，睡莲的可能性大多了）。

> 陈春成古文天赋这个何解？

我前几天读了一些关于《夜晚的潜水艇》的评论以及他自己的访谈，里面他提到他以前拿自己写的古诗词去“请教”中文系教授，结果人家教授都信以为真，还以为真的是古人写的。他还谈到他很喜欢文言文的语感，我记得他提到了一个“之”字，说有音乐感。现在我懒得回去找那篇访谈的链接了。《竹峰寺》的文字很有古韵，这句不算很有代表性，不过我现在也没时间去细寻了：

> 闽中、闽西两大山带斜贯而过，为全省山势之纲领

正好我前年[写故事时](https://yihui.org/cn/kids/2019/11/tale/#%E5%A4%A9%E4%BD%BF%E8%AF%BA%E6%81%A9)还查了一下福建的地形，但要我来写的话，我不会写出这种文言味。我又想起司马中原在评析朱西甯时还曾批评过他早年的文白混杂问题，但陈春成这篇小说的文风比较浑然一体，没有混杂的不适感。

**Reply by earfanfan** [reply]
益辉啊，我忽然发现我做错了一件事情。现在想起去年的事，我感到最遗憾的是为什么不能做到全都解释清楚、达成人类之间的互相理解呢？我忽然发现其实那个时候 tc 君发出的质疑也是问号，是问号，是问号，是我看到他的质疑的那一瞬间被剧烈波动的情绪击中，所以一下子假定了他的恶意，正是因为是我先假定了他的恶意，所以才导致后面无论如何都不能解释清楚吧，唉，就那样越解释误会越多了。

**Reply by earfanfan** [reply]
其实偶尔也有一些时刻，我对你的文字的第一反应也可能是假定恶意的，具体哪些我记不清了，不过所有的第二反应都是基于我相信你，所以可以全都变成假定善意的。

**Reply by yihui** [reply]
> 不过所有的第二反应都是基于我相信你，所以可以全都变成假定善意的

全剧终。分歧都撑不过第二集。

**Comment by earfanfan** [comment]
益辉啊，你提到的有顾虑的问题，也正是我会为你担心的一点。因为当你把文字从键盘上敲出来展示在公共场合以后，别人要怎么解读你的文字是你根本无法控制的，你也不可能要求所有阅读你的文字的人都假定善意。而且基于我去年的经验，一旦别人假定了你的初衷是恶意，那基本就是绝无可能解释得清的了。在互相信任的人之间，解释可以拨开云雾见青天，而其他情况极有可能是徒劳的、无力的。

再回到我们对《醋栗》的解读，很显然我俩之间存在分歧和误解，不过我还没捋清楚在表面的分歧之下存在哪些误解，但我觉得可以先放着，我一点也不担心误解不会解不开。所以我得先说说你对张秋子老师的误解，而你之所以会误解她当然是我的锅，因为她的思考是很长一段，但我只截取了片面。

这里需要你仔细看一下我在楼上截取的她写的片段，你看哈，她提到契诃夫是不是太苛刻了用的是问号，是问号，是问号，她并没有对契诃夫是否苛刻下结论，而她下结论的陈述句是那一段的结尾，她说她自己需要警惕打压平庸日常带来的精神优越感。

我对张秋子老师的文字的理解是这样的：当一个人在批判另一个人的平庸日常时，是否在潜意识里默认被批判的人的生活在批判人的眼里是平庸无意义的，平庸麻木的人是不是确实就是没有精神世界的？那么一个自认为有精神追求的人在批判一个被自己认为没有精神追求的人时是否会产生什么优越感呢，觉得有精神追求是带着自我实现的意义，而没有精神追求只追求物质是比前者低劣的呢……

她的思考和质疑与你的观点并不冲突，你说幸福的人在享受幸福时应该看见别人的苦难，这是我认同的，而张秋子老师的思考并不是你的观点的反面，是另一个维度。

最后，我要向你发出新的嘲讽，这个嘲讽大约有一百吨，可能会锤得你吐出一口老血。你是不是对张秋子老师的文字假定恶意了呢？如果你读了契诃夫的小说以后，很喜欢这位大师，会试图理解他不会轻易质疑他，那么张秋子老师应该也跟你一样，一样可能会读了契诃夫的小说以后会喜欢他理解他不会轻易质疑他，就算质疑也是审慎的，在对契诃夫的质疑中同时包裹着对自身的质疑。

最后的最后，我想说，其实当我俩之间出现分歧和误解的时候，我是高兴看到的。

**Reply by earfanfan** [reply]
本来是想写到前一楼的回复中的，一没注意盖了新楼……

**Reply by yihui** [reply]
其实我去年就在我的季读中还表扬了张秋子老师，我读了她的一篇文章后很佩服她，所以我才会感到吃惊。

**Reply by CyrusYip** [reply]
> 因为当你把文字从键盘上敲出来展示在公共场合以后，别人要怎么解读你的文字是你根本无法控制的，你也不可能要求所有阅读你的文字的人都假定善意。而且基于我去年的经验，一旦别人假定了你的初衷是恶意，那基本就是绝无可能解释得清的了。在互相信任的人之间，解释可以拨开云雾见青天，而其他情况极有可能是徒劳的、无力的。

我们虽然不能控制别人怎么想，但可以让自己的文字不含恶意，说话温和。我在网上也时不时有吐槽别人的冲动，不过一般都会克制住。

我刚刚重新看了去年统计之都那个帖子，本来当时想写一下自己的感受，但是帖子那会被锁定了。

> 唉，话说统计之都的坛友是不是九成都是男性啊。竟然没有一个人跟我一样，最开始看到这个下降趋势的时候第一反应是愤怒几秒钟么？虽然说生气也没撒用，可是我觉得好像已经很久没在（国内）主流视野里见过有人讨论性别不平等的问题了。

从这里开始我就觉得要开始骂战了，于淼说的打拳就是提示可能出现这种状况。我看到这里的感受是作为男性莫名其妙受到了指责，好像我很冷漠。男性也有情绪，也会难过。根据我网络冲浪的经验，这很容易开始吵架，我就委屈一下不说话了，以免点燃战火。虽然我觉得你在后面说了你是平静的情况下写的，但这些话我看了心里不舒服。这不是你平静与否的问题，是文字本身就伤人。

> 另，我最初看到三水大人写说到底份额只是从不到40%降到30%多的时候，也产生了愤怒的情绪，当时在心中腹诽“这是什么钢铁直男才说得出口的冷漠混账话啊”。

「钢铁直男」和「冷漠混账话」也很刺耳，也许于淼不在意，但其他男性可能听了不舒服。最好还是别贴标签。我刚上大学那会就被说过直男，现在想起还是感觉不舒服，凭什么我要被扣帽子啊。你在这两个地方表达了情绪和吐槽男性，帖子就从理性转向情绪了，后面有 tc 君以情绪回应也正常。情绪对情绪，帖子就越来越歪了。

我觉得那个帖子最好就只讨论数据问题，讨论完可以另开一个帖子温和地表达情绪。

**Reply by earfanfan** [reply]
叶寻同学，我理解你的感受，也明白你是善意的交流，不过我现在看到这些还是感觉好难受，虽然我都快三十一岁了，可是当下又忍不住想哭唧唧了，等我缓几天缓过来了再与你交流，好嘛

**Reply by CyrusYip** [reply]
好啊。

**Reply by XiangyunHuang** [reply]
我想楼主对「不平等」有很多兴趣，关于男女、收入等，不妨写一篇比较理性的文章。我之前找材料的时候，遇到一本书《Pareto Distributions》（帕累托分布），其中第四章讲不平等的度量，如 Gini 系数、洛伦兹曲线。书，我其实没看，楼主要是兴趣比较高，可以找一个数据集，用里面的方法分析一下。

**Reply by earfanfan** [reply]
@CyrusYip
叶寻同学，其实你的留言我看到后当时心里就有一种观点，使我犹豫的是，我觉得你才大学毕业没多久，经历社会的毒打还比较少，从我这个年纪比你大很多岁的人的角度看，你能将很多事情看得乐观积极是很好的。

不过我缓了这么久，终于决定说出本来所想的。我的观点是，人们相互交流时使用的语言是否温和、所探讨的话题是否敏感、是否怀揣善意去看待别人的言语、以及是否对别人产生了误解，每一点都是很复杂的。这些“是”和“否”分别组合在一起会得到很多种不同的情况。但“以情绪对情绪也正常”可能隐含的一个小点是，当“你”觉得别人的言语不够温和或者具有攻击性的时候，“你”可以随意审判这个人。

基于这种悲观论调，我大概会在个人博客里面继续写想写的东西，但在多数公共场合会越来越多地保持沉默。

**Reply by CyrusYip** [reply]
@earfanfan 

> 这些“是”和“否”分别组合在一起会得到很多种不同的情况。

说到这个我总会联想到线上和线下的差异。在网上我会看到有人回复的帖子恶意满满，但其实楼主并没有说错话。线下一般陌生人之间很客气，不会无端展示恶意。

匿名属性大大降低了人作恶的成本。线下骂人可能被反骂、打架、邻里关系变差。线上就可以骂人走人，留下挨骂者独自伤心。

最近就在 v2ex 看到一个恶意满满的回复。[有个28岁男子喜欢上大六岁的女子](https://www.v2ex.com/t/959145)，结果第一条评论是「不先调查下有没有老公吗」，还有73个感谢。真的不理解为什么要说那么难听的话，居然还有那么多人点感谢，太狠了……

> 基于这种悲观论调，我大概会在个人博客里面继续写想写的东西，但在多数公共场合会越来越多地保持沉默。

我觉得用太多社交网站真的是损害心理健康，太多充满恶意的人了。幸好你还有可以写的地方，个人博客气氛好很多。

> 但“以情绪对情绪也正常”可能隐含的一个小点是，当“你”觉得别人的言语不够温和或者具有攻击性的时候，“你”可以随意审判这个人。

我还没注意这个细节，感谢提醒。网络公审是严重的网络暴力。希望我以后发言能继续保持冷静。

我最近在想，有了互联网匿名这件隐身衣，我们为啥还要保持善良呢？我没有自己原创的答案，目前就照搬[佛教的因果论](https://zh.m.wikipedia.org/zh-cn/因果论_(佛教))——善有善报，恶有恶报。

**Reply by earfanfan** [reply]
@CyrusYip

>有了互联网匿名这件隐身衣，我们为啥还要保持善良呢？

其实你在思考这个问题的时候，或者说思考所有宏大命题的时候，都可以试着不要那么快给出判断或结论，毕竟越是宏大的话题，越是务虚，牵涉的范围越广，影响因素也越多。另外，关于善恶的问题，我之前看到过一种令我大开眼界的观点，意即“善”与“恶”的边界是随着时代变化的，而每个时代“善”与“恶”的定义极有可能是人为给定，至于是谁给定的，我最先能想到的是政府在教导民众时会给定一些较为清晰的以何为善以何为恶的定义，其次能想到的是一旦网络上出现某些极端恶性事件后比如某些无辜人士被网暴去世、体育馆倒塌致使十数人丧命，这种时候人们会开始尝试反思到底哪里出了问题，善与恶的边界会滑动出现新的可能性，再次就是也许我们每个人亲身经历的事情也会改变我们自己对善恶的定义。

现在国内好多社交 APP 都在推行去匿名化，改成实名制，我之前还看过一篇长篇报道，大概是有个女孩去取快递被造谣出轨快递小哥，她很笃定地反击，最终通过法律手段得到了胜利。我始终觉得科技发展太快，使得整个时代的人都被卷入一种空前的迷茫之中，但是并不是每个人都放弃了寻找方向，还是有很多人在经历以及付出努力，时代会一直变的，尽管有些部分在变坏，但一定也有变好的部分的。至少那个通过法律维护了自身权益的女孩给了很多遭受同样痛苦的人新的希望。

另外 v2ex 那个链接我打不开，不过只是看“不先调查下有没有老公吗”这句话，我倒觉得没撒恶意。

**Reply by CyrusYip** [reply]
> 其实你在思考这个问题的时候，或者说思考所有宏大命题的时候，都可以试着不要那么快给出判断或结论，毕竟越是宏大的话题，越是务虚，牵涉的范围越广，影响因素也越多。

嗯，我还是慢慢思考吧。

> 另外 v2ex 那个链接我打不开，不过只是看“不先调查下有没有老公吗”这句话，我倒觉得没撒恶意。

我认为那个女子也是单身。不知道是不是像你说的，我经历的社会毒打太少，常常往积极方面想。v2ex需要科学上网才能看，我在这贴出来吧。第二条评论是「她老公同意吗？」，你觉得这恶意吗？后面还有很多评论，我就不方便一一贴过来了。

> 然后一起学习西点培训认识的，她在我后面一组，一开始没有特别关注的...平平无奇的人
>
> 要考试，群里问有没有要拼车的，然后命运齿轮开始转动
>
> 一开始没想过找比自己年纪大的，所以相亲包括软件都是比较偏向比自己小的，但是和她的相处真的蛮舒服的，不像同龄的要背景，要房子，给的就是压力，这点从她那没感受到，这是其一
>
> 其二就是聊天的状态没有太多包袱，而且她笑点蛮低的，目前不上上课，线下见过一次，靠前图书馆复习，考试当天，包括昨天吃了个饭。
>
> 卧槽，她真的超级温柔，苏州的潮汕牛肉火锅，她推荐了一个炒饭，因为我减肥，好久没吃这种，结果爆香，一个人炫了一个汤碗那么大的量，因为考试我请过客，所以她请客了牛肉锅，下去结账时，还让师傅给我打包一份，让我冷冻带回家...
>
> ....妈的，好稀罕这种感觉
>
> 想表白，正好 8 月七夕，计划月初唠嗑拉手，七夕送金戒指表白。
>
> 本来不确定要不要发...噢！但内心澎湃，需要点赞激励下

**Reply by earfanfan** [reply]
讲真，我不想去揣度一个人一句话的动机是出于善意还是恶意。但是如果刚好我的朋友像你贴的帖子的楼主一样给我讲了这样一个故事，我也会提醒他或她要确认一下喜欢的人是不是单身。

当然啦，如果你看到那些评论第一反应是觉得不舒服，那也很正常啊。每个人会受到情绪冲击的点都不大一样。互联网就是这么一个各种各样人类的混合，会藏污纳垢，也会包容很多现实无法包容的东西。

**Reply by liechi** [reply]
@CyrusYip
> 你在这两个地方表达了情绪和吐槽男性，帖子就从理性转向情绪了......

你这里提到的也是我当时想告诉狗主的。在后来的讨论中没有给狗主递梯子还有另外一个跟当时的讨论不直接关联的原因（也是我之后在她博客下面重提此事的原因），最近我也告诉她了。虽然是想以某种方式帮她，但看起来却是比其他人对她更苛刻了。

**Reply by earfanfan** [reply]
@liechi

哈哈哈，既然列弛都说到这个份上了，其实我有个槽想吐，三水没有因为我说他是说冷漠混账话的钢铁直男而生气，大概是因为这是一段话的前半段，吐槽完他以后还有说自己想明白了心里又很感动，后面三水还给我留言叫我不要自我感动……

列弛你的“看起来更苛刻”，当时我就理解，因为你在我心里的定义就是诤友，而且你对自己要求严格，如果把我当朋友才会对我也要求严格。只不过，当时那几波冲击叠加的能量早就远超我的承受能力上限了，那对当时的我来说是个超纲题，后面千难万难也难以保持理智，所以当时是理解，但是听不到心里去……

**Reply by earfanfan** [reply]
@liechi

话又说回来，虽然知道当人与人之间交流发生分歧时，先反思自己的问题更有可能得到更和谐的结果。可是我现在好像很难做到这点，很多时候下意识第一反应就是挑别人的错处。除非是我信任的人，可以在第二反应克制一下，不然好难改正这种不知道是不是本能的东西

**Reply by liechi** [reply]
嗯，稍微克制下比较好。毕竟信任的人是少数，而需要打交道的人很多。
